src/main/java/org/testng/annotations/IListenersAnnotation.java

```diff
@@ -0,0 +1,10 @@
+package org.testng.annotations;
+
+import org.testng.ITestNGListener;
+
+public interface IListenersAnnotation extends IAnnotation {
```

Comment:
Because `IListeners` is located into the `internal` package.
There is already some "duplication" between the `annotation` package and `internal.annotation` one: I've just follow the current pattern :)